### PR TITLE
fix(storage): converts token address to lower case

### PR DESCRIPTION
**Pull request by jurajpiar**
We should be storing token addresses with lower case.

### CODE REVIEW

File: src/services/storage/handlers/offer.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ const handlers: { [key: string]: Function } = {
     }
   },
   async BillingPlanSet ({ returnValues: { period, price, token } }: EventData, offer: Offer, offerService: OfferService): Promise<void> {
-    await updatePrices(offer, period, price, token)
+    await updatePrices(offer, period, price, token.toLowerCase())
```
Comment (nduchak) [inline src/services/storage/handlers/offer.ts:98]: hmm.. we have an inconsistency in the cache and contract then, should we also lower case token address when creating a transaction to the storage contract?

Comment (jurajpiar) [reply src/services/storage/handlers/offer.ts:98]: No because we should allow users to use the contract with either format.

**Review by AuHau — APPROVED** [review]
LGTM

**Comment by AuHau** [comment]
If you would feel fancy, you could write a closure that the events pass through before reaching the handler and you could specify which properties should be lowercased like:
```
toLowerCaseAttributes(..., ['tokenAddress', ...])
```

**Review by nduchak — APPROVED** [review]

**Comment by jurajpiar** [comment]
This is being addressed [elsewhere](https://github.com/rsksmart/rif-marketplace-cache/pull/423/files#diff-cabdeead4c1efc53f99c6dc1689e8d67493999c352724eaada23f25eccac5bd1R107).

**Pull request closed without merging**